src/ORM/Table.php

```diff
@@ -1503,7 +1503,9 @@ public function get($primaryKey, array $options = []): EntityInterface
         foreach ($key as $index => $keyname) {
             $key[$index] = $alias . '.' . $keyname;
         }
-        $primaryKey = (array)$primaryKey;
+        if (!is_array($primaryKey)) {
+            $primaryKey = [$primaryKey];
```

Comment:
@markstory If the idea to use `json_encode()` was to cater for datetime primary keys then that didn't work either with current code. Casting the primary key as done earlier caused the datetime object to be converted to it's array representation and caused a `InvalidPrimaryKeyException` to be thrown below :slightly_smiling_face:.

Reply:
Well that theory is bogus then. Json was probably easy to use at the time :smile: